homeassistant/components/fritz/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: still in async_setup_entry, needs to be moved to async_setup
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: one coverage miss in line 110
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: todo
+    comment: include the proper docs snippet
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name:
+    status: todo
+    comment: partially done
+  runtime-data:
+    status: todo
+    comment: still uses hass.data
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters:
+    status: todo
+    comment: add the proper configuration_basic block
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates:
+    status: todo
+    comment: not set at the moment, we use a coordinator
+  reauthentication-flow: done
+  test-coverage:
+    status: todo
+    comment: we are close to the goal of 95%
+
+  # Gold
+  devices: done
+  diagnostics: done
+  discovery-update-info: todo
+  discovery: done
+  docs-data-update: todo
+  docs-examples: done
+  docs-known-limitations:
+    status: exempt
+    comment: no known limitations, yet
+  docs-supported-devices:
+    status: todo
+    comment: add the known supported devices
+  docs-supported-functions:
+    status: todo
+    comment: need to be overhauled
+  docs-troubleshooting: done
+  docs-use-cases:
+    status: todo
+    comment: need to be overhauled
+  dynamic-devices: done
+  entity-category: done
+  entity-device-class: done
+  entity-disabled-by-default: done
+  entity-translations: done
+  exception-translations: done
```

Comment:
It also looks like the `UpdateFailed` exception in coordinator also needs a `translation_key`

Reply:
at the moment of this PR, I wasn't aware that the UpdateFailed is translate-able 🙈 
Will add the translation later this day 👍

Reply:
tbh i'm not sure, what i could translate in this case - there is defined error message :thinking: 
```python
raise UpdateFailed(ex) from ex
```

Reply:
The second exception has an example https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/exception-translations#example-implementation -- Which seems to be similar to [what other integrations do](https://github.com/home-assistant/core/blob/076a351ce4f98d3c986ee8d2254759a4d701217a/homeassistant/components/tedee/coordinator.py#L109)

Reply:
-> #131466